app/controllers/listener/lines_controller.rb

```diff
@@ -6,13 +6,17 @@ def check
       reply_token = event['replyToken']
       rumor       = event['message']['text']
 
-      ReplyWorker.perform_async(reply_token, rumor) if rumor
+      ReplyWorker.perform_async(reply_token, rumor) if rumor and probably_forwarded?(rumor)
     end
 
     head :ok
   end
 
   private
+  
+  def probably_forwarded? rumor
```

Comment:
prefer
```suggestion
   def probably_forwarded?(rumor)
```

and, just asking, how about `forwardable?(rumor)`?

Reply:
其實取名 `probably_forwarded?` 是取「可能是（別人轉傳來的）轉傳訊息」的意思

改成 `forwardable?` 感覺好像「是否可以 forward 給 Cofacts」

意思不一樣，但作為 method name 似乎更短 + 好理解 XD

等等會一併改掉

Reply:
個人理解應該是forward給ReplyWorker, 不過英文`replyable?`沒這個字XD

Reply:
@MrOrz you got it, @tomohung 我也在想 replyable 這個字（雖然反正我們都在 rspec 裡面用一堆 truthy 了）XD